Cesium.CodeGen/Ir/TopLevel/FunctionDefinition.cs

```diff
@@ -220,6 +221,47 @@ private bool ValidateMainParameters()
             instructions.Add(Instruction.Create(OpCodes.Call, freeArgv));
         }
         instructions.Add(atExitLdLocExitCode);
+        instructions.Add(Instruction.Create(OpCodes.Call, exit)); // exit(exitCode)
+        instructions.Add(Instruction.Create(OpCodes.Ldloc_S, exitCode));
+        instructions.Add(Instruction.Create(OpCodes.Ret));
+        return syntheticEntrypoint;
+    }
+
+    private MethodDefinition GenerateSyntheticEntryPointSimple(
+        TranslationUnitContext context,
+        MethodReference userEntrypoint)
+    {
+        var syntheticEntrypoint = new MethodDefinition(
+            "<SyntheticEntrypoint>",
+            MethodAttributes.Public | MethodAttributes.Static,
+            context.TypeSystem.Int32)
+        {
+            Parameters =
+            {
+                new ParameterDefinition("args", ParameterAttributes.None, context.TypeSystem.String.MakeArrayType())
```

Comment:
For the "simple" variant, we don't need the arguments, do we?

Reply:
not sure, #87 right now handle with "simple" signature, but I'm looking at unification of "simple" and array in some way. At least this is my current understanding.

Reply:
Let's either do this unification, or drop the `args` from here :)

Also, even if you unify the code, I don't think that `args` will contribute to that.

Reply:
I'm not in mood for unification. Would like to sleep well, knowing that this PR lands. So I remove `args` from `Simple`

Reply:
No problemo.